torchvision/models/vision_transformer.py

```diff
@@ -733,6 +733,7 @@ def vit_l_32(*, weights: Optional[ViT_L_32_Weights] = None, progress: bool = Tru
 
 
 @register_model()
+@handle_legacy_interface(weights=("pretrained", ViT_H_14_Weights.DEFAULT))
```

Comment:
Same as above. Here we could put `None` but this will lead to a "no weights available" exception.